### PR TITLE
feat(api): add role mange-worker-model + rbac Handler

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#18383.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#18387.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#18388.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘

File: engine/api/v2_rbac.go [hunk 1]
```diff
@@ -54,3 +124,332 @@ func (api *API) postImportRbacHandler() ([]service.RbacChecker, service.Handler)
 			return service.WriteMarshal(w, req, nil, http.StatusCreated)
 		}
 }
+
+func (a *API) FillRBACWithNames(ctx context.Context, r *sdk.RBAC) error {
```
Comment (richardlt) [inline engine/api/v2_rbac.go:128]: This looks like loader func in dao, can we move this functions on a dedicated struct that can be RBACLoader, this struct can also contains the map used to cache groups, users, etc.

**Review by richardlt — CHANGES_REQUESTED** [review]

**Review by yesnault — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

File: sdk/rbac.go [hunk 2]
```diff
@@ -30,10 +31,10 @@ type RBAC struct {
 	Name         string         `json:"name" db:"name"`
 	Created      time.Time      `json:"created" db:"created"`
 	LastModified time.Time      `json:"last_modified" db:"last_modified"`
-	Globals      []RBACGlobal   `json:"globals" db:"-"`
-	Projects     []RBACProject  `json:"projects" db:"-"`
-	Regions      []RBACRegion   `json:"regions" db:"-"`
-	Hatcheries   []RBACHatchery `json:"hatcheries" db:"-"`
+	Globals      []RBACGlobal   `json:"globals,omitempty" db:"-"`
```
Comment (fsamin) [inline sdk/rbac.go:34]:
```suggestion
	Global      []RBACGlobal   `json:"global,omitempty" db:"-"`
```

**Review by fsamin — CHANGES_REQUESTED** [review]

**Review by fsamin — APPROVED** [review]

**Pull request merged**